build.sc

```diff
@@ -159,7 +169,8 @@ object scalalib extends MillModule {
 
   def ivyDeps = Agg(
     ivy"org.scala-sbt:test-interface:1.0",
-    ivy"org.scalameta::scalafmt-dynamic:2.0.0-RC6"
+    ivy"org.scalameta::scalafmt-dynamic:2.0.0-RC6",
+    ivy"org.scala-sbt::zinc:1.2.5"
```

Comment:
Zinc doesn't belong here either. It's currently loaded in a separate classloader, so multiple versions of Zinc can co-exist peacefully, not to mention its zoo of upstream dependencies